src/EVM/Exec.hs

```diff
@@ -44,6 +44,7 @@ vmForEthrunCreation creationCode =
     , allowFFI = False
     , freshAddresses = 0
     , beaconRoot = 0
+    , minMemoryChunk = 1
```

Comment:
Do we need this controllable? We can just set it to a sane default instead? I'd rather not have config options for stuff that a good default works for. It's useful to have this during figuring out what a good default is, but later, it may become a distraction perhaps? What do you think?

Reply:
My concern that led me to make this a tunable was symbolic execution. I'm not particularly familiar with how it is represented though so take this with a grain of salt, but what would happen if you end up having a large concrete memory due to the "optimistic growth", which then turns symbolic due to some other reason? How would that get encoded into Z3 et al? Would it hurt performance?

If my concern is unfounded I'm all for picking a good default and not having to have this option 😄